src/main/java/appeng/items/tools/powered/ToolWirelessTerminal.java

```diff
@@ -19,14 +19,27 @@
 package appeng.items.tools.powered;
 
 
+import java.awt.*;
 import java.util.EnumSet;
 import java.util.List;
 
+import appeng.api.util.AEColor;
+import appeng.client.render.BaseBlockRender;
+import appeng.client.render.blocks.RenderBlockSkyCompass;
+import appeng.client.render.items.ToolBiometricCardRender;
+import appeng.client.render.items.ToolWirelessTerminalRender;
+import appeng.core.localization.PlayerMessages;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'appeng.core.localization.PlayerMessages'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AUselessImportCheck)